src/app/services/consumer_service.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+from app.clients.client_base import ClientBase
+from app.clients.client_exception import ClientException
+from app.models import CurrencyPair, Exchange
+from app.services.exchanges.exchange_service_base import ExchangeServiceBase
+from sqlalchemy.orm.exc import NoResultFound
```

Comment:
Grouping local and external imports

```suggestion
from datetime import datetime
from sqlalchemy.orm.exc import NoResultFound

from app.clients.client_base import ClientBase
from app.clients.client_exception import ClientException
from app.models import CurrencyPair, Exchange
from app.services.exchanges.exchange_service_base import ExchangeServiceBase
```